### DISCUSSION TITLE
Job Searching!

### DISCUSSION THREAD

**Discussion by UPANOV** [community/community#54073] [Programming Help]
Good morning,

my name is Carlos Campo, 27 and fluent in Spanish and English, and know a little bit of German. I would like to get a job part-time as a developer, and I was wondering if you guys could answer some of my questions please!

. I'm looking for quick certifications that I can earn online, paid if necessary, although my budget is not so high, in order to learn how to code, both front and back end, or anything that is well know nowadays and that will give me the chance to find a job fast. Do you have any good recommendations?

. Where are the best ways to find  job in this field? I'm looking for a part-time job mainly. Do you have any recommendations on sites that would help me to do so?

. Which technologies do you think I should go for, that are more demanded?

Thanks in advance for your time, and wishing you a great day ahead!

Carlos.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mohammad-zolghadr** [comment]
Hello my friend, have a good time and I hope you are well

I will tell you the answers to the questions according to my own experiences

1. In response to your first question that you are looking for a certificate, I will introduce you to the [Coursera website](http://coursera.org/)
If you are not familiar with this site, I suggest you visit it because it has different courses that you can participate in and even get a certificate for free.
For example, I got two Certificate from the Meta  and one **Certificate** from **University of Michigan**

2. In answer to your second question, the best way to get a job is to have good **portfolios** and **resumes** and **networking**
I think that if you don't have LinkedIn, install it and be active so that more people know you and see your skills.
The world is networking world bro :)

3. These days, there are many specializations that you can earn money from, but in my opinion, **front-end development** and especially **React.js** and **TailwindCss** are among the latest and most demanded technologies.

I tried to answer your question briefly and usefully, but if you still have questions, you can ask me through the communication channels in my profile.
I am happy to help
